AndroidNativeKotlinTemplate/app/src/main/java/com/salesforce/androidnativekotlintemplate/MainActivity.kt

```diff
@@ -62,11 +70,6 @@ class MainActivity : SalesforceActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
-        // Check for and use the intent's QR code login URL if applicable.
-        if (isQrCodeLoginIntent(intent)) {
```

Comment:
Having this check in on-create was fine for just deep-linked QR codes, but it works better in on-resume for other cases such as returning from the QR code scan activity.